src/cljs/athens/electron.cljs

```diff
@@ -214,161 +216,122 @@
-       Args : db-path : type str"
+      "Remove the selected db from db-list. Update local storage value with the new db list"
       (let [current-db-list (:db-picker/all-dbs db)
             new-db-list     (into [] (filter
-                                      (fn [db-list-item]
-                                       (not= db-path
-                                             (:path db-list-item)))
-                                      current-db-list))]
+                                       (fn [db-list-item] (not= db-path (:path db-list-item)))
+                                       current-db-list))]
         {:db       (assoc db :db-picker/all-dbs new-db-list)
          :dispatch [:local-storage/set-db-picker-list]})))
 
 
   (reg-event-fx
     :db-picker/move-db
     (fn [_ [_ previous-path new-path]]
-      "Move db from current location.
-       Args : previous-path new-path : type str"
+      "Move db from current location."
       {:dispatch-n [[:db-picker/remove-db-from-list previous-path]
                     [:db-picker/add-new-db new-path]]}))
 
   (reg-event-fx
     :db-picker/select-new-db
-    (fn [{:keys [db]} [_ db-path synced]]
+    (fn [{:keys [db]} [_ db-path synced?]]
       "Select a new db from db list.
-      If the selected db is deleted from disk then show an alert
-      describing the situation and remove this db from db list.
-      Prevent selecting a db when sync is happening, instead show
-      an alert describing the situation.
-      Args :
-       filepath : type str : Filepath of db which is selected for
-                             switch.
-       synced   : type bool"
-      (if (and db-path (.existsSync fs db-path))
-         (if synced
-           {:dispatch-n [[:db/update-filepath db-path]
-                         [:boot/desktop]]}
-           (js/alert "Database is saving your changes, if you
-                     switch now your changes will not be saved"))
-       (do
-         (js/alert "This database does not exist, removing it from
-                   list")
-         {:dispatch [:db-picker/remove-db-from-list db-path]}))))
+      If the selected db is deleted from disk then show an alert describing the
+      situation and remove this db from db list. Prevent selecting a db when sync
+      is happening, instead show an alert describing the situation."
+      (let [file-exists? (and db-path (.existsSync fs db-path))]
+        (cond
+          file-exists? (cond
+                         synced? {:dispatch-n [[:db/update-filepath db-path]
+                                               [:boot/desktop]]}
+                         :else   {:dispatch  [:alert/js "Database is saving your changes, if you switch now your changes will not be saved"]})
 
+          :else        {:dispatch-n [[:alert/js "This database does not exist, removing it from list"]
+                                     [:db-picker/remove-db-from-list db-path]]}))))
```

Comment:
I did not mean to make a nested `cond` from nested `if` lol.

I meant something like

```
(cond 
  (and file-exists synced?) {}}
  (and file-exists (not synced?) {})
  :else {})
```

In otherwords, nested `if` or nested `cond` is hard to read (for me), so I tend to flatten into a flat `cond` and use `and`.

Reply:
Ok I will fix it